packages/flutter/test/widgets/overscroll_indicator_test.dart

```diff
@@ -355,13 +355,122 @@ void main() {
 
     expect(find.text('First sliver 1'), findsNothing);
 
-    await slowDrag(tester, const Offset(200.0, 200.0), const Offset(0.0, 300.0));
+    await slowDrag(tester, const Offset(200.0, 200.0), const Offset(0.0, 5.0)); // offset will be magnified ten times
```

Comment:
The `slowDrag` will ten times the offset, so just drag 50.0 pixels can trigger the overscroll indicator.

This all looks fine however this method occurs about 18 times in this test and the `slowDrag` parameter is always small value. Probably best to bow to convention and do the same here. :)
![image](https://user-images.githubusercontent.com/61075224/91410613-be52b680-e879-11ea-920a-f1fecb07e912.png)